app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -136,30 +133,24 @@ private void onConnected(GoogleSignInAccount googleSignInAccount) {
         Log.d(TAG, "Into on connected method");
         if (savedSignedInAccount != googleSignInAccount) {
             savedSignedInAccount = googleSignInAccount;
-            // Get the playerId from the PlayersClient
+            // Get the playerId from the PlayersClient.
             PlayersClient playersClient = Games.getPlayersClient(this, googleSignInAccount);
             playersClient.getCurrentPlayer()
-                    .addOnSuccessListener(new OnSuccessListener<Player>() {
-                        @Override
-                        public void onSuccess(Player player) {
-                            mPlayerId = player.getPlayerId();
-                            Log.d(TAG, "Player name : " + player.getDisplayName());
-                            switchToGameScreen();
-                            if (isActiveConnection)
-                                isActiveConnection = false;
-                        }
+                    .addOnSuccessListener(player -> {
+                        mPlayerId = player.getPlayerId();
```

Comment:
would you mind renaming mPlayerId to playerId?

Reply:
Fixed.